test/success/hlevels.prl

```diff
@@ -0,0 +1,74 @@
+Def HasAllPathsTo (#C,#c) = [(-> [c' : #C] (path [_] #C c' #c))].
+Def IsContr (#C) = [(* [c : #C] (HasAllPathsTo #C c))].
+Def IsProp (#A) = [(-> [a b : #A] (path [_] #A a b))].
+Def IsSet (#A) = [(-> [a b : #A] (IsProp (path [_] #A a b)))].
+
+Thm InhPropIsContr(#l:lvl) : [
+ (->
+  [ty : (U #l kan)] // can I use pre here?
+  [h : (IsProp ty)]
+  [a : ty]
+  (IsContr ty))
+] by [
+  unfold IsProp;
+  lam ty, h, a. {use a,lam x. `($ h x a)}
+].
+
+Thm PropPi(#l:lvl) : [
+ (->
+  [tyA : (U #l kan)]
+  [tyB : (-> tyA (U #l kan))]
+  [h : (-> [x : tyA] (IsProp ($ tyB x)))]
+  (IsProp (-> [x : tyA] ($ tyB x))))
+] by [
+  unfold IsProp; lam tyA, tyB, h, f, g.
+  <i> lam x. `(@ ($ h x ($ f x) ($ g x)) i)
+].
+
+Thm PropSet(#l:lvl) : [
+  (->
+   [tyA : (U #l kan)]
+   [h : (IsProp tyA)]
+   (IsSet tyA)
+  )
+] by [
+  // unfold IsSet, IsProp; // why can't I chain unfolds like I can for lam?
```

Comment:
I think the notation for this is `unfold [x,y,z]`. I can't remember why...

Reply:
The reason was that commas were (are?) used for something else. Think about `[unfold A, B]`.

Reply:
@favonia ahh yes, that's right!

Reply:
Ok, this is quite confusing for the user... How about only using `,` to repeat a tactic so that I could write `lam x, y, z` or `unfold x, y, z`?  If you implement a rewriting tactic `,` could be used for that as well so that I can write `rewrite H1, H2, H3`.

Alternatively there could be no separator so that I just write: `lam x y z`, `unfold x y z` and `rewrite H1 H2 H3`. Maybe the parser is not flexible enough?

Reply:
@mortberg  The problem is that we have the tactical `[t1, t2, t3]` already; what do you propose to do instead?

Reply:
Hmm, what does that do?

I kind of like the no separator syntax as it is even faster to type. :)